### ISSUE TITLE
choosing the frequency with a double-click + alphanumeric numbers keys

### ISSUE THREAD

**Issue by trebmuh** [brummer10/XTuner#4]
I found my self "naturally" trying to double-click on the "440" (initial) base frequency to be able to change it using the keyboard numbers key.
I doesn't seem to work yet, and I suggest here to add this.

**Comment by brummer10** [comment]
I didn't use double click at all. I've setup my environment so that a single click match.
However, I've implemented a pop-up spin-box on double click now, but no number key-support for now.

**Comment by brummer10** [comment]
I forgot to mention, the easiest way to set the ref-freq is by just click on it and move the mouse /up/down, for fine setting use the mouse wheel.

**Comment by trebmuh** [comment]
> I forgot to mention, the easiest way to set the ref-freq is by just click on it and move the mouse /up/down, for fine setting use the mouse wheel.

Ahhhhh ok! I didn't get that yet!
It's fine.

> [...] no number key-support for now.

Do you plan to code it in at some point? I'd be great for specific case I think, but "_click-on-it + move the mouse_" and "_mouse wheel_" uses are fine for the vast majority of uses I'd guess.

**Comment by brummer10** [comment]
> Do you plan to code it in at some point? 

Not really. Doing that with xlib isn't as simple as it looks when you use it in a regular toolkit.

**Comment by trebmuh** [comment]
OK
